edb/server/bootstrap.py

```diff
@@ -451,14 +485,6 @@ async def _make_stdlib(
     std_plans.append(schema_version)
     plan.generate(current_block)
 
-    stdglobals = '\n'.join([
```

Comment:
I don't think this is correct.  If you remove this, the global schema would not have the correct record for the default superuser at bootstrap time (as well as the devmode schema cache).